src/main/java/calculator/domain/Input.java

```diff
@@ -9,26 +9,25 @@
 
     private Input() {}
 
-    public static String[] getUserInput() throws IOException {
+    public static String getUserInput() throws IOException {
         BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
 
         String userInput = br.readLine().trim();
+        try {
+            Validator.checkIsBlack(userInput);
+        } catch (IllegalArgumentException e){
+            System.out.println(e.getMessage());
+            return null;
+        }
```

Comment:
```suggestion
       Validator.checkIsBlack(userInput);
```
예외를 던저 주기만 하는 것은 어떨까요? 
가독성을 생각한다면, null를 리턴해 `continue`로 처리하는 것보다 예외를 던지는 것이 더 나은 표현인 것 같습니다.